spec/fixtures/update_feature_event.json

```diff
@@ -96,6 +96,7 @@
         "id": "61280369",
         "name": "id",
         "value": "dummy_trello_card_id",
+        "integration_id": "dummy_trello_integration_id",
```

Comment:
this integration id allows the trello code to identify an existing integrated feature (preventing re-creation of the card during update spec run)